packages/amplify-graphql-schema-generator/src/__tests__/mysql-datasource-adapter.test.ts

```diff
@@ -95,7 +95,7 @@ describe("testDataSourceAdapter", () => {
 
     model = new Model("Country");
     const countryIdField = new Field("id", { "kind": "NonNull", "type": { "kind": "Scalar", "name": "Int" } });
-    countryIdField.default = { kind: "DB_GENERATED", value: "uuid()" };
+    countryIdField.default = { kind: "DB_GENERATED", value: "(uuid())" };
```

Comment:
Just now checked on a MySQL instance, COLUMN_DEFAULT shows as `uuid()` without the brackets? Does it differ between MySQL versions?

<img width="1000" alt="image" src="https://user-images.githubusercontent.com/83315412/214935713-f029439c-cfbc-419e-9643-2cdf831ea928.png">

Reply:
I've double checked this one and its not related to MySQL version difference. The MySQL doc meant that in the table DDL we need to have these computed values wrapped in `()` like for example, `DEFAULT (RAND())`. But when we query the table info as we do in above snippet, it trims away the `()` if it's a single function and retains if it's more complex, for example, `(RAND() * RAND())`. So, in short I need to update the method that deduces if it's a computed default to handle such cases.